pom.xml

```diff
@@ -63,7 +63,11 @@
             <artifactId>validation-api</artifactId>
             <version>1.1.0.Final</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-validator</artifactId>
+            <version>5.4.2.Final</version>
+        </dependency>
```

Comment:
I'm not sure if this is the right approach. Do you know that happens if I add `service-validator` with this dependency in a project that already has another JSR-303 implementation (e.g. a full-fledged application server)? If not let me know so I can run some tests myself and post the results. Maybe to fail at the absence of a implementation (properly helping the developer in a friendly and intuitive way via console output) is better than pushing an implementation behind the scenes?